tests/Unit/ExampleTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
Add some test cases here with the permissions to see that your code works.

Maybe do some integration tests.

Reply:
done